browser-test/src/support/applicant_questions.ts

```diff
@@ -493,6 +502,14 @@ export class ApplicantQuestions {
     await this.clickSubmit()
   }
 
+  async downloadFromConfirmationPage() {
+    // Assert that we're on the review page.
+    await this.expectConfirmationPage()
+
+    // Click on download button.
+    await this.clickDownload()
```

Comment:
**suggestion (blocking):** this clicks the download button but makes no assertion that the download succeeds. Please add that assertion using the playwright Download API: https://playwright.dev/docs/downloads